### ISSUE TITLE
[Accessibility][Sourcing]: Drag drop table rows not possible using keyboard

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#9137]
#### Is this a bug, enhancement, or feature request?
Bug

#### Briefly describe your proposal.
User is not able to perform drag and drop of rows using keyboard
This is currently possible only using mouse and is blocker for disabled users

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
Angular 13.x.x
   "@fundamental-ngx/core": "0.35.4",
    "@fundamental-ngx/platform": "0.35.4",

#### If this is a bug, please provide steps for reproducing it.
Try to drag and drop a fdp-table row using keyboard

#### Is there anything else we should know?

Blocker for guided sourcing customers

**Comment by droshev** [comment]
@I543348 What is the expectation here? Maybe the solution of that is to have a toolbar menu offering moving a specific row to a specific position?

**Comment by I543348** [comment]
Hi @droshev ,
Yes somehow a user who is not able to use the "mouse" should be able to achieve drag / drop of rows using a keyboard shortcut or a toolbar button to cut / copy / paste etc.

Thanks.

**Comment by droshev** [comment]
@I543348 We would need to discuss that request with A11Y experts how this should be done. @mikerodonnell89 will bring it in the a11y office hours(you can join that call too)

**Comment by I543348** [comment]
Thanks @droshev.

Sure I can join the call.
Request you to please add our UI architect as well - siddhartha.shrivastava@sap.com in the call.

Thanks again.

**Comment by mikerodonnell89** [comment]
We discussed some ways of implementing this and one idea would be to allow the user to focus a row or cell, then press "shift" (or some other key) to enter drag-and-drop mode. Then the arrow keys would do the dragging and dropping, and the user can press shift again to exit drag-and-drop mode. Waiting to hear back from UX designers if this is the route they'd like to see
